### PR TITLE
Adds a bunch of material to the pred ship and slightly adjusts layout to compensate.

### PR DESCRIPTION
**Pull request by s5nt**
# About the pull request

Adds a bunch of material to the pred ship including more runed sandstone, normal sandstone, more metal, along with new items that spawn in pred storage which are stacks of wood, stacks of carpet tile, candle boxes, desk lamps, lanterns, gambling table parts, fancy wooden table parts, wooden table parts, poor wooden table parts, etc. Also adds more pred specific food to the pred kitchen for lodge decoration.
Slightly adjusts pred ship layout to compensate, moving bracer storage to the other side to accommodate more racks for materials. Screenshot attached.

# Explain why it's good for the game

Allows preds to build larger and more impressive lodges (barely anybody builds lodges anyways), which provides more opportunity for RP and fun marine/xeno interactions with lodges rather than them just blowing into a room with a C4 and that being the entire lodge. Pred WL is self regulating in terms of stupidly/obstructively large lodges via pred council, obstructive lodges are essentially never an issue anyway.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

image

</details>

# Changelog

:cl:
add: Added more materials to pred ship including more sandstone, wood, table parts, etc for lodge construction, along with more pred specific food to the pred kitchen.
maptweak: Slightly expanded pred ship material storage area by moving bracer storage over to the left side of the ship.
/:cl:

### CODE REVIEW

**Comment by s5nt** [comment]
Not a very experienced mapper and I'm not sure what I'm meant to do about the "Some checks were not successful", it's like the xbox red ring of death.

**Comment by Drulikar** [comment]
> Not a very experienced mapper and I'm not sure what I'm meant to do about the "Some checks were not successful", it's like the xbox red ring of death.

They were caused by you interrupting already running checks with a merge commit and github being janky about ordering for actions.

File: maps/predship/huntership.dmm [hunk 1]
```diff
@@ -1118,6 +1142,7 @@
 /area/yautja)
 "cC" = (
 /obj/structure/machinery/autolathe/yautja,
+/obj/structure/closet/wardrobe/chaplain_black,
```
Comment (Nanu308) [inline maps/predship/huntership.dmm:1145]: what is this and why is it below the autolathe?

**Review by Nanu308 — CHANGES_REQUESTED** [review]

File: maps/predship/huntership.dmm (hunk 1 above)
Comment (s5nt) [reply maps/predship/huntership.dmm]: fixed, removed

**Review by Nanu308 — APPROVED** [review]
mapping lgtm

**Pull request merged**